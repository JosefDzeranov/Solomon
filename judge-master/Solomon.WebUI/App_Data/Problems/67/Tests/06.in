100
3 126 3
47 52 3
33 1 3
138 18 3
52 3 3
33 140 3
149 2 3
32 1 3
174 1 3
30 2 3
123 76 3
113 19 3
97 63 3
16 165 3
104 3 3
129 1 3
3 3 3
94 2 3
49 32 3
16 1 3
40 12 3
7 1 3
11 85 3
128 43 3
72 56 3
14 96 3
138 3 3
16 58 3
35 142 3
66 2 3
103 1 3
123 1 3
108 24 3
86 48 3
75 24 3
101 83 3
111 83 3
59 7 3
42 1 3
68 7 3
33 91 3
108 2 3
14 2 3
33 35 3
111 1 3
85 1 3
160 31 3
134 2 3
92 15 3
99 2 3
16 157 3
16 150 3
68 10 3
131 1 3
136 7 3
107 1 3
39 145 3
11 2 3
97 2 3
47 2 3
188 3 3
72 2 3
78 3 3
75 1 3
81 50 3
49 124 3
35 4 3
126 2 3
45 2 3
96 92 3
132 1 3
92 88 3
88 4 3
104 4 3
77 1 3
143 4 3
68 75 3
128 1 3
42 17 3
92 2 3
124 35 3
101 93 3
22 8 3
47 102 3
39 1 3
110 1 3
19 2 3
2 1 3
16 112 3
113 4 3
133 1 3
85 81 3
101 2 3
110 64 3
34 1 3
1 1 3
59 100 3
107 67 3
6 49 3
35 77 3
