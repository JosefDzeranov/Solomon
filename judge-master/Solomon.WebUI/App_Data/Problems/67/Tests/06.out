156
52
96 88 17 22 58 43 20 87 83 10 8 3 95 67 85 39 69 60 5 38 30 62 64 75 63 46 73 80 18 59 50 93 31 15 56 42 86 45 90 32 68 78 16 54 71 91 48 27 76 7 9 61 